account_block_account/models/account_block_mixin.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class AccountBlockMixin(models.AbstractModel):
+    """Inherit this from models that can be blocked"""
+    _name = 'account.block.mixin'
+    _description = 'Mixin for accounting entities that can be blocked'
+
+    blocked = fields.Boolean(default=False)
+
+    @api.multi
+    def name_get(self):
+        return [
+            (this_id, '%s%s' % (
+                name, '' if not self.browse(this_id).blocked else ' [X]'
```

Comment:
Wouldn't it be better to place the [X] at the front so that in case of long accounting lines, people will still notice the X? In that case we may also prepend one of [these](https://stackoverflow.com/questions/8086375/what-character-to-use-to-put-an-item-at-the-end-of-an-alphabetic-list) characters to make sure that they come at the end of the list.

Reply:
Particularly funny would be †

Reply:
I wanted to do the cool unicode thing of printing https://en.wikipedia.org/wiki/Right-to-left_mark and then do fancy stuff about it, but not all browsers like this.
But the outcome of `name_get` doesn't influence sort order, in fact, it just processes the outcome of `search`

Reply:
Ok clear.. I would still prefer the X in front though, and maybe inheriting modules could implement name search to do the ordering.